e2e/cypress/e2e/organization/organizations.cy.ts

```diff
@@ -1,4 +1,35 @@
+import { renameOrg } from 'support/api/orgs';
+import { apiAuth } from '../../support/api/apiauth';
+
+const orgPath = `/org`;
+const testOrgNameChange = 'e2erenametest';
+
 describe('organizations', () => {
+  describe('rename', () => {
+    beforeEach(() => {
+      apiAuth().as('api');
+      cy.visit(orgPath);
+    });
+
+    afterEach(() => {
+      apiAuth()
+        .as('api')
+        .then((api) => {
+          renameOrg(api, 'ZITADEL');
+        });
+    });
```

Comment:
because its used for permission.cy.ts later and old ZITADEL name is required.. or would you change it to an alternate name first an the test to 'ZITADEL'?

Reply:
In beforeEach, I'd ensure a separate org for this test via API and ensure it doesn't have the target name.

I think ensureOrg already exists?

With the current implementation, you can't be sure that the cleanup code is actually run.

Reply:
change it to an other approach.. beforeeach is now ensuringanorg and the test is switching to the new org and renaming it